draft-irtf-cfrg-ristretto.xml

```diff
@@ -39,14 +38,14 @@ a prime-order group.</t>
 <t>Ristretto is a technique for constructing prime-order groups with
 non-malleable encodings from non-prime-order elliptic curves.  It
 extends the <xref target="Decaf"></xref> approach to cofactor elimination to support
-cofactor-8 curves such as Curve25519 <xref target="RFC7748"></xref>. In particular, this
-allows an existing Curve25519 library to provide a prime-order group
+cofactor-8 curves such as Curve25519 <xref target="RFC7748"></xref>. In particular,
+this enables a prime-order group using an existing Curve25519 library
```

Comment:
Similarly, I think the original phrasing is better because it's more explicit: the existing Curve25519 library can provide a prime-order group.  If we just say "this enables a prime-order group using an existing Curve25519 library" then it could be read as saying that a *user* of any existing Curve25519 library can use Ristretto, which is not the case, since most Curve25519 libraries (rightly) don't expose the underlying field arithmetic that would be required to build the abstraction layer.